Loop/View Controllers/CarbEntryViewController.swift

```diff
@@ -285,9 +321,15 @@ final class CarbEntryViewController: LoopChartsTableViewController, Identifiable
             break
         }
     }
-
+    
     override func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
-        return NSLocalizedString("Choose a longer absorption time for larger meals, or those containing fats and proteins. This is only guidance to the algorithm and need not be exact.", comment: "Carb entry section footer text explaining absorption time")
+        var footerMessage = ""
+        if shouldDisplayAccurateCarbEntryWarning {
+            footerMessage.append(NSLocalizedString("Your Blood Glucose is rising. Please make sure your carb entry is aligned with the time you eat.\n\n", comment: "Warning to ensure the carb entry is accurate"))
```

Comment:
[nit] Why is "Blood Glucose" capitalized? Is this consistent with the rest of the app?

Also, why the two newlines at the end?

Also, is the "aligned with the time you eat" the approved copy? (It seems like it would be confusing. Perhaps something like "Your Blood Glucose is rising. Please make sure the date of your carb entry is the same as when you actually ate."

Reply:
I believe this is placeholder copy. Sorry I should have include a comment about that.